### DISCUSSION TITLE
What are Skipped mutants?

### DISCUSSION THREAD

**Discussion by martinssipenko** [infection/infection#1658] [Q&A] [answered]
When I'm running Infection in CI I see that there are several Skipped mutants but I don't quite get what that means. I also was not able to find this information in the documentation. Could someone explain what are Skipped mutants in more detail?

Also, when I run Infection locally I do not get any skipped mutants, any ideas what could be the reason for that? Locally I have the exact same setup as on CI.

**Comment by maks-rafalko** [comment, marked as answer]
Please see the following explanation on Infection docs site: https://infection.github.io/2020/08/18/whats-new-in-0.17.0/#Skip-S-mutations-that-are-over-specified-time-limit

> Also, when I run Infection locally I do not get any skipped mutants, any ideas what could be the reason for that?

Probably, tests are executed slower on CI and on CI configured `timeout` (which is 10 seconds by default) is not enough, thus you have `S`kipped Mutants.

Try to increase this value and check if you still have Skipped Mutants on CI.

**Reply by martinssipenko** [reply]
Thanks, I've updated the timeout value and mutations are not being marked as skipped anymore.
